.github/workflows/consuming.yml

```diff
@@ -100,8 +100,29 @@ jobs:
       - name: Make sure ${{ matrix.project }} is using the built Shipyard image
         run: sed -i 's/shipyard-dapper-base:*.*/shipyard-dapper-base:dev/' ${{ matrix.project }}/Dockerfile.dapper
 
-      - name: Run all linters
-        run: make -C ${{ matrix.project }} lint
+      - name: Run gitlint
+        if: always()
+        run: make -C ${{ matrix.project }} gitlint
```

Comment:
Since they all basically perform the same action now `make -C ${{ matrix.project }} <linter>` can't we just add to the matrix: `linter: ['gitlint', 'golangci-lint', 'markdownlint', 'packagedoc-lint', 'shellcheck', 'yamllint']`?

Then we can simply call `make -C ${{ matrix.project }} ${{ matrix.linter }}` :)

This also avoids the need for awkward `if: always()` ;)